### ISSUE TITLE
FAM ID in family navigator

### ISSUE THREAD

**Issue by WGroleau** [vesta-webtrees-2-custom-modules/vesta_classic_laf#120]
Is it possible for this ID to b the ID of the family instead of the spouse's ID?

Family navigator
Family with Maria Kackley (**P3884**) ←←
himself  	Amos Spaid (P3894)

wife 
Maria Kackley (P3884)
1807–1875
![temp](https://user-images.githubusercontent.com/5910274/216462225-1332d9af-2141-4764-a324-18c67d0570b4.jpg)

**Comment by ric2016** [comment]
No, anyway not easily - See the description in the control panel: "Display a family's XREF after the family label."
This sidebar doesn't show the family label (which would be "Amos Spaid + Maria Kackley"), and therefore no family Xref is shown here.

I do not plan to extend this functionality (actually it may not have been a good idea to include it at all in the first place).
